image-hotspot-question.js

```diff
@@ -371,6 +432,11 @@ H5P.ImageHotspotQuestion = (function ($, Question) {
 
     // Clear feedback
     this.removeFeedback();
+
+    // Reset timer
+    if (this.isRoot()) {
+      this.setActivityStarted(true);
```

Comment:
Good thought! Other content types don't do this however, so I'm not sure if this should be introduced here.

Reply:
True, It does not seem to require here!